### ISSUE TITLE
Proxy `PUT` and `DELETE` methods fail for json-ld without type in root object

### ISSUE THREAD

**Issue by Laurin-W** [activitypods/activitypods#273]
Here, `jsonld.parser.expandTypes` fails, if I want to remove a permission from a resource using the semapps authProvider's `removePermission` method.

https://github.com/activitypods/activitypods/blob/b403dce369d8b46408522aa5b556e53727ba77da/packages/core/middlewares/app-control.js#L60-L69

The `types` value is empty and the error is raised here: https://github.com/assemblee-virtuelle/semapps/blob/48d8cf431b0421f69491e145fc7bea4736158ec9/src/middleware/packages/jsonld/services/parser/index.js#L89-L92

The json in the `PUT` request looks like this:
```json
{
  "@context": {
     ...
  },
  "@graph": [
    {
      "@id": "#Read",
      "@type": "acl:Authorization",
      "acl:accessTo": "http://localhost:3000/t1/data/1e858645-d48d-490f-a705-482178c4f0c9",
      "acl:agentClass": [],
      "acl:mode": "acl:Read"
    }
  ]
}
```

Should I change `app-control.js:66` to: ` resource.type || resource['@type'] || arrayOf(resource['@graph']).map(resource => resource['@type'] || resource.type)` ?

**Comment by srosset81** [comment]
Right, there should be a special handling of the `acl:Authorization` type. Ideally we should ensure that the user as a `acl:Control` permission on the linked resource (checking for a `acl:Write` permission on the ACL authorization will not work).
